.travis.yml

```diff
@@ -1,23 +1,23 @@
 language: python
 python:
-  - 2.7_with_system_site_packages
```

Comment:
so, `with_system_site_packages` thing forces travis to use whatever python is shipped with distribution. IIRC it's 2.7.6 at the moment. Then there were some problems installing opencv... It was easier to go with virtualenv and install packages from pip instead (reason to upgrade from cv/cv2 to cv3)

Reply:
This is a nice improvement.